scripts/qa/generate_sus_op_reconciliation_report.ps1

```
@@ -0,0 +1,382 @@
+    $summarySheet = $workbook.Worksheets.Item(1)
+    $summarySheet.Name = 'Executive Summary'
+    $yearSheet = $workbook.Worksheets.Item(2)
+    $yearSheet.Name = 'Year Summary'
+    $detailSheet = $workbook.Worksheets.Item(3)
+    $detailSheet.Name = 'Provider Detail'
+    $sqlSheet = $workbook.Worksheets.Item(4)
+    $sqlSheet.Name = 'SQL Used'
+
+    $navy = 0x6B351F
+    $blue = 0xD9A23B
+    $lightBlue = 0xF3E5D3
```

PSScriptAnalyzer (1.25.0)

[warning] The variable 'lightBlue' is assigned but never used.

(PSUseDeclaredVarsMoreThanAssignments)